requirements/requirements-linux.txt

```diff
@@ -1,34 +1,34 @@
-astroid==2.2.3
+astroid==2.2.2
 beautifulsoup4==4.7.1
 bs4==0.0.1
 certifi==2018.11.29
 chardet==3.0.4
 Click==7.0
 gTTS==2.0.3
 gTTS-token==1.1.3
-idna==2.8
-isort==4.3.10
+idna==2.5
```

Comment:
any particular reason for changing the versions?

Reply:
There was conflict on the package version requirement by old packages and new ones needed for the further functions.